### DISCUSSION TITLE
mikrotik

### DISCUSSION THREAD

**Discussion by zstergios** [ZoeyVid/cloudflared#91] [Q&A]
The original tunnel code is:
cloudflare/cloudflared:latest tunnel --no-autoupdate run --token xxxxxx

I replaced the code on mikrotik with:

zoeyvid/cloudflared:latest --no-autoupdate run --token xxxxxx

Seems is running but not working.

What's the command I should use?

The reason I am using your docker is that supports ARM x86

**Comment by Zoey2936** [comment]
try `zoeyvid/cloudflared:latest tunnel run --token xxxxxx`

**Comment by zstergios** [comment]
Thanks, I am not sure if it's your issue the log says

`2024-04-19T15:17:29Z ERR update check failed error="Get \"https://update.argotunnel.com?arch=arm&clientVersion=2024.4.0&os=linux\": dial tcp: lookup update.argotunnel.com: i/o timeout"`

https://prnt.sc/xb-e-fsEABbQ
https://prnt.sc/oIhg4VBG_qvo

MAYBE can't reorganize the token

EDIT:

There is an issue there
https://github.com/cloudflare/cloudflared/issues/388
